.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,9 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Please remove all `.idea` file from the commit. In general, I am not a fan of listing different IDEs in our [`.gitignore`](https://github.com/strongloop/loopback-next/blob/master/.gitignore), but considering the popularity of JetBrain IDEs, it may be a sensible thing to do.

@raymondfeng @strongloop/loopback-maintainers what's your opinion? We are keeping VS Code settings inside our git repo, should we perhaps promote JetBrains IDEs like IDEA and WebStorm as supported IDEs and version their configuration files too?

See also https://github.com/strongloop/loopback-next/blob/master/docs/site/WEBSTORM.md

Reply:
I tend to keep IDE settings inside our project, it would be very friendly for beginners to build/test/debug.